packages/webapp/.env

```diff
@@ -1,5 +1,7 @@
-NEXT_PUBLIC_API_URL=http://api.local.com
-#NEXT_PUBLIC_API_URL=https://api.daily.dev
+#NEXT_PUBLIC_API_URL=http://api.local.com
+NEXT_PUBLIC_API_URL=https://api.daily.dev
+
+
```

Comment:
Once we have verified the transition to be working well, I think we can then revert the change of the env variables 🚀